halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/aws/AwsAddAccountCommand.java

```diff
@@ -62,6 +67,46 @@ protected String getProviderName() {
       required = true)
   private String assumeRole;
 
+  @Parameter(
+      names = "--launching-lifecycle-hook-default-result",
+      description = AwsCommandProperties.HOOK_DEFAULT_VALUE_DESCRIPTION)
+  private String launchingHookDefaultResult = "ABANDON";
+
+  @Parameter(
+      names = "--launching-lifecycle-hook-heartbeat-timeout-seconds",
+      description = AwsCommandProperties.HOOK_HEARTBEAT_TIMEOUT_DESCRIPTION)
+  private Integer launchingHookHeartbeatTimeoutSeconds = 3200;
```

Comment:
Here the default is 3200 specifically.  If this should be 3600 (one hour), needs to change.